pkgs/by-name/pa/packetry/package.nix

```
@@ -0,0 +1,48 @@
+{
+  fetchFromGitHub,
+  lib,
+  stdenv,
+  rustPlatform,
+  gtk4,
+  pkg-config,
+  pango,
+  wrapGAppsHook4,
+  darwin,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "packetry";
+  version = "0.2.2";
+
+  src = fetchFromGitHub {
+    owner = "greatscottgadgets";
+    repo = "packetry";
+    rev = "v${version}";
```

Helps with branch/tag confision.

```suggestion
    rev = "refs/tags/v${version}";
```